README.md

```diff
@@ -684,28 +684,29 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 ### Collaboration and Team Tools
 
 * [Adium](https://adium.im/) - Free instant messaging application for Mac OS X. Connect to AIM, MSN, SMPP, Yahoo and more. ![Freeware][Freeware Icon]
+* [Bluesky](https://bsky.app/) - Bluesky is social media as it should be. Find your community among millions of users, unleash your creativity, and have some fun again. [![Open-Source Software][OSS Icon]](https://github.com/bluesky-social) ![Freeware][Freeware Icon]
```

Comment:
It is a web application without a macOS client.

Reply:
I know, but they have an iOS version for Apple Silicon Macs. Since they are a startup, if they ever develop a macOS version, I’ll add it to the list.

Reply:
Our project is a list of macOS applications. Including command-line tools was already a stretch, but now adding applications that don’t even have a client interface is downright unreasonable.

Reply:
OK